device.c

```diff
@@ -139,6 +139,7 @@ struct iio_channel * iio_device_find_channel(const struct iio_device *dev,
 			continue;
 
 		if (!strcmp(chn->id, name) ||
+				(chn->name && !strcmp(chn->label, name)) ||
```

Comment:
Shouldn't this be ... (chn->label && !strcmp(chn->label, name)) ...

Reply:
Yes, it should. Thanks!